docs/guides/requests_storage.md

```diff
@@ -0,0 +1,137 @@
+---
+id: requests-storage
+title: Requests Storage
+---
+
+The Apify SDK has several requests storage types that are useful for specific tasks. The requests are stored either in a run memory (applies to Request List only, see below), on local disk to a directory defined by the
+`APIFY_LOCAL_STORAGE_DIR` environment variable, or on the [Apify platform](/docs/guides/apify-platform) under the user account identified by the API token defined by the `APIFY_TOKEN` environment variable. If neither of these variables is defined, by default Apify SDK sets `APIFY_LOCAL_STORAGE_DIR` to `./apify_storage` in the current working directory and prints a warning.
+
+Typically, you will be developing the code on your local computer and thus set the `APIFY_LOCAL_STORAGE_DIR` environment variable. Once the code is ready, you will deploy it to the Apify platform, where it will automatically set the `APIFY_TOKEN` environment variable and thus use cloud storage. No code changes are needed.
+
+**Related links**
+
+-   [Apify platform storage documentation](https://docs.apify.com/storage)
+-   [View storage in Apify app](https://my.apify.com/storage)
+-   [Request queues API reference](https://docs.apify.com/api/v2#/reference/request-queues)
+
+## Request queue
+
+The request queue is a storage of URLs to crawl. The queue is used for the deep crawling of websites, where you start with several URLs and then recursively follow links to other pages. The data structure supports both breadth-first and depth-first crawling orders.
+
+Each actor run is associated with a **default request queue**, which is created exclusively for the actor run. Typically, it is used to store URLs to crawl in the specific actor run. Its usage is optional.
+
+In Apify SDK, the request queue is represented by the [`RequestQueue`](/docs/api/request-queue) class.
+
+In local configuration, the request queue data is stored in the directory specified by the `APIFY_LOCAL_STORAGE_DIR` environment variable as follows:
+
+```
+{APIFY_LOCAL_STORAGE_DIR}/request_queues/{QUEUE_ID}/{STATE}/{NUMBER}.json
+```
+
+Note that `{QUEUE_ID}` is the name or ID of the request queue. The default queue has ID `default`, unless you override it by setting the `APIFY_DEFAULT_REQUEST_QUEUE_ID` environment variable. Each request in the queue is stored as a separate JSON file, where `{STATE}` is either `handled` or `pending`, and `{NUMBER}` is an integer indicating the position of the request in the queue.
+
+The following code demonstrates basic operations of the request queue:
+
+```javascript
```

Comment:
I would get rid of this example and replace it with something more useful. No one is ever using request queue like that.

Reply:
Done